cluster.py

```diff
@@ -0,0 +1,333 @@
+import boto3.session
+import requests
+import os.path as path
+import logging
+import json
+import collections
+import re
+import textwrap
+import time
+import os
+from functools import total_ordering
+
+LOG = logging.getLogger(__name__)
+
+PEERS_FILE = os.getenv('PEERS_FILE', '/etc/sysconfig/etcd-peers')
+
+CLIENT_SCHEME = os.getenv('CLIENT_SCHEME', 'http')
+PEER_SCHEME = os.getenv('PEER_SCHEME', 'http')
+
+CLIENT_PORT = int(os.getenv('CLIENT_PORT', 2379))
+PEER_PORT = int(os.getenv('PEER_PORT', 2380))
+
+MAX_RETRIES = int(os.getenv('MAX_RETRIES', 10))
+
+HOSTED_ZONE_ID = os.getenv('HOSTED_ZONE_ID', None)
+DOMAIN_NAME = os.getenv('DOMAIN_NAME', None)
+
+boto = None
```

Comment:
Want to hear Your opinion. For me global variable isn't solution. Maybe object creation should be handled by constructor and boto could be passed as argument?